store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -170,6 +176,75 @@ public CarbonReaderBuilder filter(Expression filterExpression) {
     return this;
   }
 
+  public CarbonReaderBuilder filter(String columnName, String value) {
```

Comment:
this always creates equal to filter, so keeping filter as name is not good.

I suggest change it to **`public static Expression prepareEqualToExpression(String columnName, String value)`**

so get the equalto filter expression from above and then pass it to existing filter() method. This way it can support for all the filter expression.

Reply:
done